### PR TITLE
2022q1 accessibility initial commit

### PR DESCRIPTION
**Pull request by pauamma**
Taking off my status hat and making this a pull request instead
so I can have a review before merging it.

### CODE REVIEW

**Review by debdrup — COMMENTED** [review]
I noticed only one minor nit.

File: 2022q1/accessibility.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+== FreeBSD Accessibility
+
+Links
+link:https://wiki.freebsd.org/Accessibility[Accessibility wiki page] URL: link:https://wiki.freebsd.org/Accessibility[https://wiki.freebsd.org/Accessibility] +
+Link: link:https://lists.freebsd.org/archives/freebsd-accessibility/2021-October/000000.html[List introduction, goals, audience, and ground rules] URL: link:https://lists.freebsd.org/archives/freebsd-accessibility/2021-October/000000.html[link:https://lists.freebsd.org/archives/freebsd-accessibility/2021-October/000000.html] +
```
[the remark this answers is not in the document]
Comment (pauamma) [reply 2022q1/accessibility.adoc]: Removed

It comes straight from report-sample.adoc. Any reason it should be kept there?

[the remark this answers is not in the document]
Comment (debdrup) [reply 2022q1/accessibility.adoc]: Nope, not that I can think of. I'll go fix it.

**Comment by alfonsosiciliano** [comment]
@pauamma this is a vary important project, I' ll try to help, in the past I received an email from an user to use https://edbrowse.org (www/edbrowse) on FreeBSD.

**Review by debdrup — APPROVED** [review]
Forgot to approve this.

**Pull request merged**